docs/deploy/2-vm.mdx

```diff
@@ -4,77 +4,113 @@ title: Deploy to a VM
-- A valid SSL certificate issued by a [certificate authority](https://en.wikipedia.org/wiki/Certificate_authority) (self-signed certs do not work here).
-- Ability to add a DNS record pointing to your new LiveKit Server.
+- A domain that you control
+- Ability to add DNS records for subdomains that point to your new LiveKit Server
+
+## Generate configuration
+
+Use our configuration generation tool to create a customized configuration for your domain.
+
+```shell
+docker run --rm -it -v$PWD:/output livekit/generate
+```
+
+You should have a folder with the following files:
+
+* caddy.yaml
+* docker-compose.yaml
+* livekit.yaml
+* redis.conf
+* init_script.sh OR cloud_init.xxxx.yaml
 
-## Deploy with Cloud Init
+## Deploy to a VM
 
-This is the easiest method for deploying LiveKit Server. You can deploy a LiveKit Server using a vanilla Linux image and Cloud Init configuration file.
+Depending on your cloud provider, there are a couple of options
 
 <Tabs
-  defaultValue="aws"
-  groupId="cloud"
+  defaultValue="cloud-init"
+  groupId="cloud-deploy"
   values={[
-    {label: 'AWS', value: 'aws'},
-    {label: 'Digital Ocean', value: 'do'},
+    {label: 'Cloud Init', value: 'cloud-init'},
+    {label: 'Startup Script', value: 'startup-script'},
   ]}>
 
-  <TabItem value="aws">
+  <TabItem value="cloud-init">
 
-LiveKit Server can be deployed with either Amazon Linux 2 or Ubuntu. Use the `cloud-init.<platform>.yaml` file corresponding to your desired platform:
+  This is the easiest method for deploying LiveKit Server. AWS, Azure, Digital Ocean, and others support [cloud-init](https://cloud-init.io/).
 
-- For Amazon Linux 2, use [cloud-init.amzn2.yaml](https://raw.githubusercontent.com/livekit/livekit-server/master/deploy/cloud-init.amzn2.yaml)
-- For Ubuntu, use [cloud-init.ubuntu.yaml](https://raw.githubusercontent.com/livekit/livekit-server/master/deploy/cloud-init.ubuntu.yaml)
+  We have tested our scripts on Ubuntu and Amazon Linux, but it's possible that the same scripts may also work on other platforms.
+  (Please let us know in Slack or open a PR!)
 
-Steps:
-
-1. Download one of the above `cloud-init.<platform>.yaml` files. Edit it to include your LiveKit Server configuration, SSL certificate, and SSL private key.
-2. Open the [AWS Console](https://console.aws.amazon.com/ec2) and navigate to EC2.
-3. Launch an instance.
-4. Choose the latest "Amazon Linux 2" or "Ubuntu Server" AMI.
-5. In the "Step 3: Configure Instance Details" screen, copy and paste text from your modified `cloud-init.yaml` file from step 1 into the "User data" field, under the "Advanced Details" section.
-6. Launch your instance as usual.
+  When starting a VM, paste the contents of the file `cloud-init.xxxx.yaml` into the `User data` field.
 
+  That's it, when the machine starts up, it'll execute the cloud-init protocol and install LiveKit
```

Comment:
That's it! When the machine starts up, it'll execute the cloud-init protocol and install LiveKit.